fetch-todolist/src/customHook/useFetch.js

```diff
@@ -1,29 +1,29 @@
 import { useState, useEffect } from "react";
+import { useQuery } from "react-query";
+import { getTodos } from "../services/todos.services.js";
 
 const useFetch = () => {
-  const [data, setData] = useState([]);
-  const [loading, setLoading] = useState(true);
-  const [error, setError] = useState("");
-  useEffect(() => {
-    fetch("http://localhost:8080/todo")
-      .then((resp) => {
-        if (!resp.ok) throw Error("Fetching To Do List Failed");
-        return resp.json();
-      })
-      .then((todolist) => {
-        const timer = setTimeout(() => {
-          setData(todolist);
-          setLoading(false);
-          clearTimeout(timer);
-        }, 1000);
-      })
-      .catch((err) => {
-        setError(err.message);
-        setLoading(false);
-      });
-  }, []);
+  const { data, isLoading, error } = useQuery("todos", getTodos);
+  // useEffect(() => {
```

Comment:
why commented code, do we need it ?

Reply:
I had kept it for my reference,as we switched from customHooks to react-Query()